src/test/java/com/project/acre/service/MemberServiceTest.java

```diff
@@ -0,0 +1,28 @@
+package com.project.acre.service;
+
+import com.project.acre.controller.MemberDto;
+import com.project.acre.util.SHA256;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.security.NoSuchAlgorithmException;
+
+public class MemberServiceTest {
+
+    @Test
+    public void joinPasswordEncrypt() throws NoSuchAlgorithmException {
+        MemberDto memberDto = new MemberDto();
+        SHA256 sha256 = SHA256.getInstance();
+        memberDto.setPassword(sha256.encrypt("test1"));
+
+        Assertions.assertThat(memberDto).extracting("password").isEqualTo(sha256.encrypt("test1"));
+    }
```

Comment:
위 테스트를 보면 조금 단순화해서 보면
sha256.encrypt("test1").equals(sha256.encrypt("test1"))
를 하는것과 동일합니다.
즉, 테스트의 역할을 제대로 못하고 있어요 (1 ==1 이 true인지 확인하는게 테스트의 의미가 있을까요?)

위와같은 경우는 expect 를 실제 hash 문자열로 테스트 하는게 좋습니다.
```suggestion
        Assertions.assertThat(memberDto).extracting("password").isEqualTo("1b4f0e9851971998e732078544c96b36c3d01cedf7caa332359d6f1d83567014"); 
    }
```

다른 검증된 sha256 툴에서 test1을 암호화 한 결과를 expect로 정의함으로서, 테스트 하고자 하는 로직 (sha256.encrypt) 의 동작을 확인할 수 있게 됩니다. 만일 누군가SHA256 구현을 잘못 변경했다면, 위 테스트가 실패하면서 위험을 감지할 수 있겠지요!!